.pre-commit-config.yaml.jinja

```diff
@@ -44,7 +57,7 @@ repos:
           - --branch={{ odoo_version }}
           - --template-filename=.module-readme.rst.j2
   - repo: https://github.com/myint/autoflake
-    rev: v1.4
+    rev: v2.2.0
```

Comment:
Should this (and the rest of libraries) be unconditionally changed for all Odoo versions?

Reply:
I've personally been updating the hook versions in this file even on projects still using odoo 12, and have not come across any particular issues when doing so. I vote in favour.

Reply:
OK then